docusaurus/docs/user-docs/intro.md

```diff
@@ -5,109 +5,50 @@ sidebar_label: Welcome!
-| --------------------- | ------------------------------------------------------------------------------------------------------- |
-| First name            | Write your first name in the textbox.                                                                   |
-| Last name             | Write your last name in the textbox.                                                                    |
-| Email                 | Write your complete email address in the textbox.                                                       |
-| Username              | (optional) Write a username in the textbox.                                                             |
-| Interface language    | Among the drop-down list, choose a language for your Strapi application interface.                      |
-| Interface mode        | Among the drop-down list, choose a mode for your Strapi application interface: either "Light mode" or "Dark mode". Note that by default, the chosen mode for a Strapi application is based on the browser's mode. |
+## What you will find here
 
-4. Click on the **Save** button.
+The table of content in this User Guide displays 7 main sections.
 
-### Changing your password
+Clicking on any of the following cards will direct you to the introductory page for that section, with additional details and concepts:
 
-To change the password of your account:
-
-1. Go to your administrator profile.
-2. Fill in the password-related options:
-
-| Password modification |                                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------- |
-| Current password      | Write your current password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
-| Password              | Write the new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown.      |
-| Password confirmation | Write the same new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
-
-3. Click on the **Save** button.
-
-
----
-
-Congratulations on being a new Strapi user! You're now ready to discover all the features and options that Strapi has to offer!
+<CustomDocCardsWrapper>
+  <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />
+  <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, Manage and Publish Content Types including collection types." link="/user-docs/content-type-builder" />
+  <CustomDocCard emoji="🗃️" title="Media Library" description="Access all assets, upload assets, configure the view, and manage folders." link="/user-docs/media-library" />
```

Comment:
I would suggest a shorter and slightly more accurate description. What do you think?
```suggestion
  <CustomDocCard emoji="🗃️" title="Media Library" description="Upload and manage all assets and organize them into folders." link="/user-docs/media-library" />
```

Reply:
Yeah this is better, thanks